src/directives/formio.js

```diff
@@ -48,7 +48,6 @@ module.exports = function() {
           'false': false
         };
 
-        var submission = $scope.submission || {data: {}};
```

Comment:
Was there a reason this got removed? I'm fine with it as long as it doesn't cause side effects but it would be good to know the reason behind it.